upstream/solr6/local/92-init-solr.sh

```diff
@@ -110,6 +110,9 @@ function makeConfigs {
         setOption 'alfresco.suggestable.property.3' '{http://www.alfresco.org/model/content/1.0}content' "$SHARED_PROPERTIES"
     fi
 
+    # Setting cross locale property for consistent behaviour with older versions ( =< 1.4.0)
+    setOption 'alfresco.cross.locale.property.2' '{http://www.alfresco.org/model/content/1.0}description' "$SHARED_PROPERTIES"
```

Comment:
Why are we writing to `alfresco.cross.locale.property.2` exactly?
It looks like an arbitrary key to set it under which may conflict with configurations by other users.

Reply:
Just FYI: configuration options in shared properties can be configured with environment variables prefixed with `SHARED_`

Reply:
I'll change the index on the property to something high and check if it works.

I would want to add the change to the image since adding the `SHARED_` envvar on every new installation doesnt seem like a good solution.

Reply:
Yes. Would be nice to know why is needed (and not needed in ASS 1.4.0).
As far as I see, there is no explicit mention of description in:

https://github.com/xenit-eu/alfred-api/blob/master/apix-integrationtests/src/main/java/eu/xenit/apix/tests/search/SearchServiceFacetsTest.java

Reply:
The test that is affected is in the superclass (https://github.com/xenit-eu/alfred-api/blob/7ea35f75826947a002a360d9957e84e9fa2a612e/apix-integrationtests/src/main/java/eu/xenit/apix/tests/search/SearchServiceTest.java#L271)
Was added by Dries last year I believe, the issue itself was not with transactional/eventual of the desc property, but with limits.

Reply:
In that case, you are papering over issues by just fixing it for the property that is tested in Alfred API.

If you want that test to succeed, you can configure it in the test docker-compose, not in this project.